azext_iot/_params.py

```diff
@@ -384,21 +386,37 @@ def load_arguments(self, _):
             help="If set, keys are exported normally. Otherwise, keys are "
             "set to null in export output.",
         )
+        context.argument(
+            "storage_authentication_type",
+            options_list=["--auth-type", "--storage-authentication-type"],
+            arg_type=get_enum_type(AuthenticationType),
+            help="Authentication type for communicating with the storage container.  "
+            "Allowed values: identity, key",
```

Comment:
I would double check if you need to provide the text of allowed values for enum types. The arg parse framework should take care of it.

Reply:
This actually may have been left behind before I added the `arg_type` and removed the `Based` suffix... You're right, this is likely unnecessary